osquery/tables/system/linux/processes.cpp

```diff
@@ -36,8 +36,8 @@ namespace tables {
 std::string proc_name(const proc_t* proc_info) {
```

Comment:
Can this whole function just be simplified to
```c++
std::string proc_name(const proc_t* proc_info) {
  return std::string(proc_info->cmd);
}
```
since it [looks like](https://gitorious.org/procps/procps/source/195fdc43654ae84c679c3ddad24ef03fe43f7977:proc/readproc.c#L81) `proc_info->cmd` is a null-terminated character array?